pending-documentation.md

```diff
@@ -1361,6 +1362,27 @@ evaluating another function, we would also be able to see the arguments of that
 function as well, unless the function was tail recursive, in which case no
 stack frame would exist once the tail call was performed.
 
+### Breaking on a set of rules or functions
+
+Using `rbreak <regex>` you can set breakpoints on multiple functions.
+
+-   `rbreak Lbl` - sets a breakpoint on all non hooked `function`s
+
+-   `rbreak Lbl.*TEST` - sets a breakpoint on all `function`s from module `TEST`
+
+-   `rbreak hook_INT` - sets a breakpoint on all hooks from module `INT`
+
+### Other debugger issues
+
+-   `Python Exception <class 'gdb.error'> No type named string`
```

Comment:
this line can be removed as we now always build the backend with debug symbols in the runtime, even in release configuration.

Reply:
It doesn't work though. Latest version from master:
```
radu@ubuntu:~/work/test$ rm -rf test-kompiled/
radu@ubuntu:~/work/test$ kompile test.k -ccopt -g -ccopt -O1
radu@ubuntu:~/work/test$ krun --output kast --debugger
(gdb) break definition.kore:step
Breakpoint 1 at 0x215310
(gdb) r
Breakpoint 1, 0x0000000000215310 in step (Python Exception <class 'gdb.error'> No type named string.: 
subject=0x155552e001a0)
```
Changed to Debug in llvm-backend/pom.xml
```
radu@ubuntu:~/work/test$ rm -rf test-kompiled/
radu@ubuntu:~/work/test$ kompile test.k -ccopt -g -ccopt -O1
radu@ubuntu:~/work/test$ krun --output kast --debugger
(gdb) break definition.kore:step
Breakpoint 1 at 0x28c2b0
(gdb) r
Breakpoint 1, 0x000000000028c2b0 in step (subject=
    `<generatedTop>{}`(`<k>{}`(`kseq{}`(Lblinit{}(.KList),`kseq{}`(Lblsmt{}(.KList),dotk{}(.KList)))),`<generatedCounter>{}`(#token("0", "Int"))))
```